server/models.py

```diff
@@ -98,16 +94,13 @@ def active(self):
 
 class Participant(db.Model, TimestampMixin):
     id = db.Column(db.Integer(), primary_key=True)
-    user = db.Column(db.ForeignKey("user.id"), index=True, nullable=False)
+    user_id = db.Column(db.ForeignKey("user.id"), index=True, nullable=False)
     course_id = db.Column(db.ForeignKey("course.id"), index=True,
                           nullable=False)
-    role = db.Column(db.Enum(*VALID_ROLES, name='role'), nullable=False)
-    course = db.relationship("Course", backref="participants")
+    role = db.Column(db.Enum(*VALID_ROLES, name='role'), default=STUDENT_ROLE, nullable=False)
 
-    def __init__(self, user, course_id, role=STUDENT_ROLE):
-        self.user = user
-        self.course_id = course_id
-        self.role = role
+    user = db.relationship("User", backref="participants")
```

Comment:
Sidenote for merge: I called the backref `participations`, enrollments would be the ideal name - but we already have a User model method named enrollments. I'll sort it out during the merge